README.md

```diff
@@ -1,61 +1,67 @@
 # stated
 
-final state
+Final state
```

Comment:
i prefer the lower case :D

Reply:
Done in 8fc558f5b6587224a0bb8c872e3a425e5046c4a3